src/app/pages/profile/transaction-information/transaction-information.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { TransactionInformationService } from '../../../services/profile/transaction-information/transaction-information.service';
+import { APPCONFIG } from 'src/app/config';
+
+@Component({
+  selector: 'app-transaction-information',
+  templateUrl: './transaction-information.component.html',
+  styleUrls: ['./sass/transaction-information.component.scss']
+})
+export class TransactionInformationComponent implements OnInit {
+  base_url = APPCONFIG.base_url;
+  transacationUrl = `${this.base_url}/transactions/`;
```

Comment:
If the approach above is taken, then this code will change to
```suggestion
  transacationUrl = `${environment.api_url}/transactions/`;
```

Reply:
worked on that, thanks